static/fluent/pl/main.ftl

```diff
@@ -0,0 +1,524 @@
+
+## Update Settings
+update-settings-update-now = Aktualizuj teraz
+update-available = Dostępna nowa wersja
+update-up-to-date = Twój system jest aktualny
+last-update = Ostatnia aktualizacja
+current-version = Obecna wersja
+failed = Niepowodzenie
+never = Nigdy
+in-progress = W trakcie
+restarting = Restartowanie
+
+## Developer Settings
+developer-settings-enable-ssh = Włącz SSH
+developer-settings-view-internal-logs = Wyświetl Dzienniki Wewnętrzne
+developer-settings-create-local-authorization = Utwórz autoryzację lokalną
+
+## Rules
+rules = Reguły
+rules-create-rule-hint = Nie utworzono reguł. Kliknij +, aby utworzyć regułę.
+rules-rule-name = Nazwa Reguły
+rules-customize-rule-name-icon =
+  .alt = Dostosuj Nazwę Reguły
+rules-rule-description = Opis Reguły
+rules-preview-button =
+  .alt = Podgląd
+rules-delete-icon =
+  .alt = Usuń
+rules-drag-hint = Przeciągnij tutaj urządzenia, aby rozpocząć tworzenie reguły
+rules-drag-input-hint = Dodaj urządzenie jako wejściowe
+rules-drag-output-hint = Dodaj urządzenie jako wyjściowe
+rules-scroll-left =
+  .alt = Przewiń w lewo
+rules-scroll-right =
+  .alt = Przewiń w prawo
+rules-delete-prompt = Upuść urządzenie tutaj, aby rozłączyć
+rules-delete-dialog = Czy na pewno chcesz trwale usunąć te reguły?
+rules-delete-cancel =
+  .value = Anuluj
+rules-delete-confirm =
+  .value = Usuń Reguły
+rule-invalid = Niepoprawna
+rule-delete-prompt = Czy na pewno chcesz trwale usunąć tę regułę?
+rule-delete-cancel-button =
+  .value = Anuluj
+rule-delete-confirm-button =
+  .value = Usuń Regułę
+rule-select-property = Wybierz Właściwość
+rule-not = Nie
+rule-event = Zdarzenie
+rule-action = Akcja
+rule-configure = Konfiguruj...
+rule-time-title = Godzina
+rule-notification = Powiadomienie
+notification-title = Tytuł
+notification-message = Wiadomość
+notification-level = Poziom
+notification-low = Niski
+notification-normal = Normalny
+notification-high = Wysoki
+rule-name = Nazwa Reguły
+rule-and = i
+rule-or = lub
+rule-time-of-day-is = jest godzina
+rule-event-lower = zdarzenie
+rule-event-occurs = ma miejsce
+
+## Logs
+logs = Wykresy
+logs-create-log-hint = Nie utworzono wykresów. Kliknij +, aby utworzyć wykres.
+logs-device = Urządzenie
+logs-device-select =
+  .aria-label = Rejestrowane Urządzenie
+logs-property = Właściwość
+logs-property-select =
+  .aria-label = Rejestrowana Właściwość
+logs-retention = Retencja
+logs-retention-length = Czas Retencji
+  .aria-label = Rejestrowany Czas Retencji
+logs-retention-unit =
+  .aria-label = Rejestrowana Jednostka Retencji
+logs-hours = Godziny
+logs-days = Dni
+logs-weeks = Tygodnie
+logs-save = Zapisz
+logs-remove-dialog-title = Usuwanie
+logs-remove-dialog-warning = Usunięcie wykresu spowoduje również usunięcie wszystkich jego danych.
+  Czy na pewno chcesz go usunąć?
+logs-remove = Usuń
+logs-unable-to-create = Nie można utworzyć dziennika
+logs-server-remove-error = Błąd serwera: nie można usunąć dziennika
```

Comment:
In all the other places "log" is translated as "wykres", which as far as I understand makes sense, because all logs are actually charts. It probably makes sense to keep using "wykres" unless this string refers to something else.

Reply:
Changed to "wykresu".